ciab.openapi.yaml

```diff
@@ -750,13 +750,13 @@ paths:
           multipart/form-data:
             schema:
               properties:
-                From:
+                dateFrom:
```

Comment:
So, did we settle on camelCase? For some reason, my brain had it that we'd settled on snake_case (and that's what I used in my store update the other day)...

Reply:
My brain has that we had not settled yet... :)  I was just making everything match for now so that when we decided it would be eaiser to come in and fix it.

So snake_case is it?